### ISSUE TITLE
Failed to extract windows vhdx. LASTEXITCODE: 2 (AksEdge.psm1:4781)

### ISSUE THREAD

**Issue by pbering** [Azure/AKS-Edge#118]
**Describe the bug**

Not possible to create new deployment of type LinuxAndWindows nor adding Windows node afterwards. This has been working on previous AKS Edge versions (before GA).

**To Reproduce**

1. Follow intructions: <https://learn.microsoft.com/en-us/azure/aks/hybrid/aks-edge-howto-single-node-deployment#create-a-mixed-workload-cluster>
2. Fails with:

```text
[05/11/2023 09:34:58] Extracting Windows Vhdx

System ERROR:
The filename, directory name, or volume label syntax is incorrect.

[05/11/2023 09:34:58] Exception Caught!!!

 - Failed to extract windows vhdx. LASTEXITCODE: 2 (AksEdge.psm1:4781)
```

**Expected behavior**

That it is possible to add a Windows node.

**Environment:**

 - AKS Edge Essentials Version: 1.1.220.0 April Release
 - Kubernetes version: 1.24.6 (k8s)
 - Windows Host OS:
    - Edition: Windows 11 Enterprise
    - Version: 22H2, OS Build 22621.1635
    - Virtual Machine: NO

**Comment by pbering** [comment]
ohh found the issue... was running in the latest PowerShell Core 7.3.4, when switching to legacy v5.0 then it works!

**Comment by fcabrera23** [comment]
Hi @pbering,

Great to see you fixed your issue. However, probably the issue was not related to PS, because we support PS7.

Thanks,
Francisco

**Comment by pbering** [comment]
Hi Francisco, 

I think this issue should remain open. The issue is in `"C:\Program Files\WindowsPowerShell\Modules\AksEdge\1.1.220.0\AksEdge.psm1"` line `4779`, if you run the following in PowerShell 7.3.4:

```
& "C:\Program Files\AksEdge\AksEdgeWindows-v1.exe" "x" "-o""C:\temp""" "-y" 
```

it fails with `The filename, directory name, or volume label syntax is incorrect.`. But if you run it in PowerShell 5 then it works and extracts fine.

If the quoting of the `-o` argment is removed so the line looks becomes:

```
& "C:\Program Files\AksEdge\AksEdgeWindows-v1.exe" "x" -o""C:\temp"" "-y" 
```

The it works across v5 and v7.

**Comment by fcabrera23** [comment]
Hi @pbering,

Closing this issue - It was fixed with [1.2.414.0](https://github.com/Azure/AKS-Edge/releases/tag/1.2.414.0) release.

Thanks,
Francisco
